amida-tech/indaba-client
# Feature/INBA-42 PM All Subjects Page

#### What's this PR do?
Implement the All Subjects view for PMs

#### Related JIRA tickets:
[INBA-42](https://jira.amida-tech.com/browse/INBA-42)

#### How should this be manually tested?
1. Log in as admin in an instance with some subjects
1. Click All Subjects to go to /subjects
1. Check that all subjects across all projects are displayed
1. Check that the search bar filters the list
1. Check that clicking the delete icon next to a subject tries and fails to delete the subject (backend call to `DELETE /uoas/:id`, 400 error from the backend, see [INBA-442](https://jira.amida-tech.com/browse/INBA-442)

#### Any background context you want to provide?
This branch sits on the #220 branch, review that first

#### Screenshots (if appropriate):
